autoload/lsp/completion.vim

```diff
@@ -263,7 +263,7 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
       d.icase = 1
     endif
 
-    if item->has_key('kind')
+    if item->has_key('kind') && !item.kind->empty()
```

Comment:
i may think it's a bug, 
'kind' should not be a 'null', and it should not be a '0' either,
you may should ask pylsp to fix it.